modules/modules.mk

```diff
@@ -94,8 +95,20 @@ ifeq ($(XFEM),yes)
         TENSOR_MECHANICS            := yes
 endif
 
+ifeq ($(SCALAR_TRANSPORT),yes)
+	CHEMICAL_REACTIONS          := yes
+	NAVIER_STOKES               := yes
+	THERMAL_HYDRAULICS	    := yes
+        FLUID_PROPERTIES            := yes
+        HEAT_CONDUCTION             := yes
+        RDG                         := yes
+        RAY_TRACING                 := yes
+        SOLID_PROPERTIES            := yes
+        MISC                        := yes
```

Comment:
```suggestion
	      CHEMICAL_REACTIONS          := yes
	      NAVIER_STOKES               := yes
	      THERMAL_HYDRAULICS	        := yes
        FLUID_PROPERTIES            := yes
        HEAT_CONDUCTION             := yes
        RDG                         := yes
        RAY_TRACING                 := yes
        SOLID_PROPERTIES            := yes
        MISC                        := yes
```

Reply:
What's with all the extra spaces here?

Reply:
I see now - weird how the rendering is different in the snippet view compared to the overview.